### ISSUE TITLE
docs(accessibility content): In-line accessibility content within component guidance

### ISSUE THREAD

**Issue by connor-leech** [carbon-design-system/carbon-website#524]
## Overview

We've been tasked with bringing existing accessibility information from a standalone website into Carbon docs. At the moment, we're bringing the content over en masse and adding a fourth tab to our component pages.

## The issue

This is the most hands-off and practical approach for an immediate solution. That being said, we need to start exploring how we can prevent these docs from blocking our ongoing efforts to consolidate component guidance into a single, navigable page.

The quantity, intent, and presentation of these docs is not aligned with our current approach to component documentation. Designers and developers using our site are not looking for an exhaustive repository of information (citation needed), but rather a scannable resource containing clear and actionable information.

Validation and research is undoubtedly a weak point for the system, but I don't believe providing it in-line is the best approach.

## Alternative options

As has been discussed in the past, we should be providing at-a-glance accessibility information for our components. Is it accessible, and to what standard?

We can achieve this through tags at the top of pages, and on a status page if needed. We can provide links to additional testing and documentation for those requiring it.

### Competitive analysis

| System | In-line docs? | Notes |
| ------- | ------------- |  ------ |
| [Carbon](https://www.carbondesignsystem.com/guidelines/accessibility/overview) | No | Carbon offers an accessibility overview page but does not offer consistent in-line accessibility guidance on component pages |
| [Quickbooks](https://designsystem.quickbooks.com/bolt/accessibility/) | No | Quickbooks offers sparse accessibility docs and no in-line guidance |
| [US Gov.](https://designsystem.digital.gov/components/button/) | Yes | The US Gov. design system offers extensive accessibility guidance both generally and in-line on component pages |
| [UK Gov.](https://design-system.service.gov.uk/accessibility/) | No | While considered a leader in the accessibility field, the UK Gov does not offer many accessibility resources in-line on component pages. They do, however, [occasionally link](https://design-system.service.gov.uk/components/date-input/) to relevant requirements. |
| [Salesforce](https://lightningdesignsystem.com/components/accordion/#Accessibility) | Sometimes | Salesforce includes relevant accessibility information on some pages, but not all. They also offer general accessibility guidance. |
| [Material](https://www.material.io/components/buttons/#text-button) | Yes | Material provides minimal but useful accessibility guidance on almost every component page. They mention the principle and link out to more resources. |
